kcqrs-client-gson/build.gradle

```diff
@@ -31,8 +31,8 @@ dependencies {
   compile 'com.google.code.gson:gson:2.8.0'
   compile 'com.google.http-client:google-http-client:1.23.0'
   compile 'com.google.http-client:google-http-client-gson:1.23.0'
-  
-  testCompile group: 'junit', name: 'junit', version: '4.12'
+
+  testCompile project(':kcqrs-testing')
```

Comment:
The idea of testing package is to be used from client apps and  doesn't look reasonable to share some common jmock utilities.

Reply:
Reverted.